sds-go/go/scanner.go

```diff
@@ -80,7 +81,9 @@ func CreateScanner(ruleConfigs []RuleConfig) (*Scanner, error) {
 			return nil, ErrInvalidKeywords
 		case -4: // rust: CreateScannerError::InvalidMatchAction
 			return nil, ErrInvalidMatchAction
-		case -5: // rust panic
+		case -5: // rust: CreateScannerError::InvalidMetadata
```

Comment:
I don't really understand why we need to expose this new error type in the Go Binding. In fact, the validation of standard rules should not have much impact on the scanner itself.